test_runner/regress/test_close_fds.py

```diff
@@ -23,7 +23,7 @@ def test_lsof_pageserver_pid(neon_simple_env: NeonEnv):
     env = neon_simple_env
 
     def start_workload():
-        env.neon_cli.create_branch("test_lsof_pageserver_pid")
+        env.create_branch("test_lsof_pageserver_pid")
```

Comment:
the whole test and I believe our only lsof dependency could be removed.

this also has an extra branch.